contracts/base/OwnerManager.sol

```diff
@@ -11,6 +11,9 @@ import {IOwnerManager} from "../interfaces/IOwnerManager.sol";
  * @author Richard Meissner - @rmeissner
  */
 abstract contract OwnerManager is SelfAuthorized, IOwnerManager {
+    // SENTINEL_OWNERS is used to traverse `owners`, so that:
```

Comment:
We also have a `SENTINEL_MODULE` for modules linked list in ModuleManager.sol, could you please add a similar comment there and then it would be good to go

Reply:
Do you think we should add it here as well?

https://github.com/safe-global/safe-smart-account/blob/main/docs/overview.md#owner-management

Reply:
> The linked list head and tail are the 0x1 address. The head and tail are never removed from the list. The head and tail are never owners.

It is somehow already mentioned there

Reply:
@mmv08 done.